### DISCUSSION TITLE
No "Permissions" tab in a group

### DISCUSSION THREAD

**Discussion by mpalavrov** [keycloak/keycloak#12029] [Archive]
I have noticed that with the **new Admin Console** there is no "Permissions" tab for the Groups:

- New Admin Console:
![image](https://user-images.githubusercontent.com/33352511/168739171-8762f4a6-6df3-4f12-bcc5-09c59af4d542.png)

- Old Admin Console:
![image](https://user-images.githubusercontent.com/33352511/168739293-257ff5df-4b9b-47df-8ea8-f91a2b5251cc.png)

Please note that in the documentation, it is still described that we can go in Permissions tab of a Group and configure it, https://www.keycloak.org/docs/latest/server_admin/#group

**Comment by ssilvert** [comment]
Hi @mpalavrov.

The tab there in the latest nightly build.  I'm still finishing fine-grained authorization, but it should be ready in a couple of weeks.

**Comment by mpalavrov** [comment]
Hi @ssilvert,
Thank you very much for looking at this and for providing a fix.
Cheers,
Milen
